include/circt/Dialect/SV/SVStatements.td

```diff
@@ -100,7 +100,7 @@ def IfDefProceduralOp
 def IfOp : SVOp<"if", [SingleBlock, NoTerminator, NoRegionArguments,
                        ProceduralRegion, ProceduralOp]> {
   let summary = "'if (cond)' block";
-  
+
```

Comment:
Plz trim out unnecessary diffs to help make the review simpler, or commit whitespace-only changes separately.